openage/convert/main.py

```diff
@@ -156,6 +156,28 @@ def convert_assets(assets, args, srcdir=None):
     conversion experience, then passes them to .driver.convert().
     """
 
+    assets = root["assets"]
+    cfg = root["cfg"]
+
+    asset_location_path = cfg / "asset_location"
+    if srcdir is None:
+        try:
+            with asset_location_path.open("rb") as file_obj:
+                srcdir = CaseIgnoringDirectory(file_obj.read().strip()).root
+                print('Found previous AOE2 install path: ' + srcdir.resolve_native_path().decode('utf-8'))
```

Comment:
If it is not utf8, this decoding needs error handling i think.

Another improvement might be the test if `srcdir` does still exists, otherwise the question doesn't make much sense :)

Reply:
I think I fixed the utf-8 decoding error handling with `errors='replace'`. I also think I made the second improvement, but am not sure.

Reply:
`CaseIgnoringDirectory` will raise a `FileNotFoundError` if the directory doesn't exist.